documentation/Rotational Motion.tex

```diff
@@ -0,0 +1,187 @@
+\documentclass[10pt, a4paper, twoside]{basestyle}
+
+\usepackage{tikz}
+\usetikzlibrary{cd}
+
+\usepackage[Mathematics]{semtex}
+\usepackage{chngcntr}
+\counterwithout{equation}{section}
+
+%%%% Shorthands.
+
+\newcommand{\squarenorm}[1]{\scal{#1}{#1}}
+
+%%%% Title and authors.
+
+\title{Rotational Motion of a Rigid Reference Frame}
+\date{\printdate{2023-04-26}}
+\author{Pascal~Leroy (phl)}
+\begin{document}
+\maketitle
+\begin{sloppypar}
+\noindent
+This document describes the computations that are performed by the class \texttt{RigidReferenceFrame} and its subclasses to determine the rotational motion (rotation,
+angular velocity, and angular accelation) of a rigid frame.
+\end{sloppypar}
+
+\section*{Definitions}
+We consider in this document a rigid reference frame defined by two bodies $B_1$ and $B_2$ at positions $\vq_1$ and $\vq_2$, respectively.  We construct a basis of the reference frame using three vectors having the following properties:
+\begin{itemize}
+\item{the \emph{fore} vector $\vF$ which is along the axis $\vq_2 - \vq_1$;}
+\item{the \emph{normal} vector $\vN$ which is orthogonal to $\vF$ and is such that the velocity of the frame, $\dot{\vq}_2 - \dot{\vq}_1$, is in the plane $\pa{\vF, \vN}$;}
```

Comment:
Remove `of the frame,`; the velocity of the frame sounds like (𝒒&#x0307;₁ + 𝒒&#x0307;₂) / 2.

Reply:
Done.